.github/workflows/auto-triage-issues

```diff
@@ -0,0 +1,20 @@
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  auto_label:
+    runs-on: ubuntu-latest
+    name: Automatic Github Issue Labeller
+    steps:
+    - name: Label Step
+      uses: larrylawl/Auto-Github-Issue-Labeller@main
```

Comment:
Why are we using their @main branch? Don't they have a release version?

Reply:
They dont

Reply:
They have v1
https://github.com/larrylawl/auto-github-issue-labeller-action/tags

Reply:
Updated it.